src/gstreamer/Util/KvsSinkUtil.cpp

```diff
@@ -66,6 +67,10 @@ gboolean parseIotCredentialGstructure(GstStructure *g_struct, std::map<std::stri
         ++it) {
         params_key_set.insert(it->first);
     }
+    if( params_key_set.count(IOT_THING_NAME) == 0) {
```

Comment:
Minor: try to adhere to the style guidelines in this file. The spacing around if and brackets are off

Reply:
fixed up the spacing